tsl/test/expected/compression_bgw.out

```diff
@@ -205,3 +205,66 @@ ERROR:  permission denied to start compress_chunks background process as role "n
 \set ON_ERROR_STOP 1
 RESET ROLE;
 REVOKE NOLOGIN_ROLE FROM :ROLE_DEFAULT_PERM_USER;
+\c :TEST_DBNAME :ROLE_SUPERUSER
+DROP TABLE IF EXISTS conditions CASCADE;
+NOTICE:  drop cascades to table _timescaledb_internal.compress_hyper_2_4_chunk
+CREATE TABLE conditions(
+    time TIMESTAMPTZ NOT NULL,
+    device INTEGER,
+    temperature FLOAT
+);
+SELECT * FROM create_hypertable('conditions', 'time', 'device', 30,
+       chunk_time_interval => '1 day'::interval,
+       migrate_data => true);
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             7 | public      | conditions | t
+(1 row)
+
+INSERT INTO conditions
+SELECT time, (random()*30)::int, random()*80 - 40
+FROM generate_series(NOW() - INTERVAL '90 days', NOW(), '10 min') AS time;
```

Comment:
can you use fixed start here instead of now otherwise this might lead to flaky tests on certain days when additional chunks get created due to boundaries

Reply:
Not sure why this would be flaky. Intention is that the test output should not be dependent on the number of chunks created anywhere, just that some are dropped (hence I have a big range, so at least one chunk should be marked as dropped).

Reply:
This is inherently flaky and should be avoided. Different months have different number of days, and during time zone changes (e.g., switching daylight savings) there are different number of hours per day. This could lead to different number of chunks being created in the end.

Reply:
The number of chunks in the table will not affect the output of the test.

Reply:
This has been a problem numerous times in the past. Changing to a constant timestamp is an easy change that is well worth it.